src/PageParser.php

```diff
@@ -277,6 +277,8 @@ public function getContent( $isMainPage ) {
 		$this->removeNodesWithXpath( '//meta[starts-with(@property, "mw:PageProp/")]' );
 		// Remove <meta typeof="mw:Includes/NoInclude"/> and <meta typeof="mw:Includes/NoInclude/End"/>
 		$this->removeNodesWithXpath( '//meta[starts-with(@typeof, "mw:Includes/NoInclude")]' );
+		// Remove mw:Extension meta and link elements.
+		$this->removeNodesWithXpath( '//meta[starts-with(@typeof, "mw:Extension")]' );
```

Comment:
Yeah I started to look into that, but got discouraged because I wasn't sure what required things would be needed (aside from styles, but that's a good point). I'll try to revisit it soon. (So far I've just been led by epubcheck complaining about specific things.)